dnscrypt-proxy/example-dnscrypt-proxy.toml

```diff
@@ -479,25 +479,25 @@ cache_neg_max_ttl = 600
 
 
 ###########################################################
-#        Pattern-based IP blocking (IP blacklists)        #
+#        Pattern-based IP blocking (IP blocklists)        #
 ###########################################################
 
-## IP blacklists are made of one pattern per line. Example of valid patterns:
+## IP blocklists are made of one pattern per line. Example of valid patterns:
 ##
 ##   127.*
 ##   fe80:abcd:*
 ##   192.168.1.4
 
-[ip_blacklist]
+[blocked_ips]
 
   ## Path to the file of blocking rules (absolute, or relative to the same directory as the config file)
 
-  # blacklist_file = 'ip-blacklist.txt'
+  # blocked_ips_file = 'blocked_ips.txt'
```

Comment:
Since the suggested log file name (line 500) is `blocked-ips.log` i feel like it might be a bit more consistent to also call this `blocked-ips.txt`. All the other filenames are also using kebab-case (`public-resolvers.md`, `service-install.bat` etc) so perhaps it would be best to stick with kebab-case for filenames and snake_case for the config options.

Reply:
thanks, this does look much more consistent now - inc in latest PR update

Reply:
Changes look great, thanks!